### PR TITLE
Flesh out day 3 schedule and (some) materials, improve anchor-ref behavior

### PR DESCRIPTION
**Pull request by MattF-NSIDC**
* Schedule and materials pages are now fully populated, but there are still some WIP/TODO materials.
    * The sentence about incomplete schedules on the schedule page is removed.
* When clicking a link to an anchor, the header is no longer in the way. The target of the link is also highlighted for 3 seconds (animated) so it's more obvious to the user what they jumped to.
* Minor CSS tweaks:
    * Fix weird vertical spacing in lists
    * Pick a color for "modules" that works well with dark and light mode (via transparency)

### CODE REVIEW

File: custom.scss [hunk 1]
```diff
@@ -2,6 +2,25 @@
...
+*/
+html {
+  scroll-padding-top: 85px;
+}
+
+
+/* Override built-in style which adds space after nested list elements */
+ul ul, ol ol, ul ol, ol ul {
+  margin-bottom: 0px !important;
+}
```
Comment (MattF-NSIDC) [inline custom.scss:20]: The old style had a bottom-margin on nested lists, so e.g.

* Foo
    * Bar
* Baz

Would display as 

* Foo
    * Bar

* Baz

File: custom.scss [hunk 2]
```diff
@@ -2,6 +2,25 @@
 
 /*-- scss:rules --*/
 
+$module-bg-color: rgba(188, 188, 254, 0.25);
```
Comment (MattF-NSIDC) [inline custom.scss:5]: This new transparent background color displays better in both light and dark mode.

File: .spelling
Comment (MattF-NSIDC) [inline .spelling:1]: Part of this change is sorting. Sorry for the noise this adds to the review :)

File: content/exercises/dataset-analysis-plan.qmd [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Dataset analysis plan
+
+## 💬 Discuss your group's ideas for analysis
+
+Come to a team agreement on the analysis you'd like to perform. Choose an
```
Comment (trey-stafford) [inline content/exercises/dataset-analysis-plan.qmd:5]: Nitpick: `you'd` -> `you would`.

**Review by trey-stafford — APPROVED** [review]
One nitpick item, otherwise looks great!

**Pull request merged**